src/TranspileError.ts

```diff
@@ -1,9 +1,8 @@
 import * as ts from "typescript";
 
 export class TranspileError extends Error {
-    public node: ts.Node;
-    constructor(message: string, node: ts.Node) {
+    public name = 'TranspileError';
+    constructor(message: string, public node?: ts.Node) {
```

Comment:
Don't know if we should make `node` optional.

Reply:
Makes sense, but it would be invalid once we'll enable `strictNullChecks`, because tests have to get error messages from TSTLErrors. The only type-safe way I see is making TSTLErrors just do string formatting. It also would make stack traces a bit neater.

```ts
export const formatCouldNotFindEnumMember = (enumDeclaration: ts.EnumDeclaration, enumMember: string) => `Could not find ${enumMember} in ${enumDeclaration.name.text}`
```